languages/js/src/dataFiltering.test.ts

```diff
@@ -35,14 +43,77 @@ class Num {
   number!: number;
 }
 
+@Entity()
+export class Issue {
```

Comment:
It would be good to include the relationships here too, to make sure there aren't any clashes between Polar's relations and the typeorm relations?

Reply:
👍